namespaces/live-1.cloud-platform.service.justice.gov.uk/money-to-prisoners-prod/resources/rds.tf

```diff
@@ -0,0 +1,42 @@
+# https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance
+
+variable "cluster_name" {}
+variable "cluster_state_bucket" {}
+
+module "rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.4"
```

Comment:
The latest version of this module is 4.6

Please bump the version number and check the example to ensure you have all the necessary entries.